scalyr_agent/log_processing.py

```diff
@@ -2417,12 +2417,10 @@ def perform_processing(self, add_events_request, current_time=None):
                     if self._log_stream:
                         import scalyr_ingestion_client.log_line as ingestion_client_line  # pylint: disable=import-error
 
-                        if not new_events_buffer:
-                            sequence_start_number = sequence_number
                         sequence_end_number = sequence_number
                         new_event_timestamp = line_object.timestamp
                         if not new_event_timestamp:
-                            new_event_timestamp = int(current_time)
+                            new_event_timestamp = int(current_time * 1000000000)
```

Comment:
Per Steven's comment on the other PR - we only want to set ``timestamp`` attribute on the ``LogLine`` if it's different from a timestamp on the previous log line.

Aka since we periodically poll for new data / lines, all the lines we read as part of a single poll will contain the same timestamp so we will only need to set it on the first ``LogLine`` object in that group.